samples/bluetooth/bap_broadcast_sink/src/main.c

```diff
@@ -831,16 +803,26 @@ static void base_recv_cb(struct bt_bap_broadcast_sink *sink, const struct bt_bap
 	printk("Received BASE with %d subgroups from broadcast sink %p\n",
 	       bt_bap_base_get_subgroup_count(base), sink);
 
+	/**
+	 * Get a 32-bit bitfield of BIS indexes that cover the channel allocation values in
+	 * CONFIG_TARGET_BROADCAST_CHANNEL.
+	 */
```

Comment:
This comment should be moved to the `#if defined(CONFIG_TARGET_BROADCAST_CHANNEL)` clause